src/main/java/seedu/address/model/person/PersonEvent.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.person;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class PersonEvent {
+    private final int date;
+    private final int month;
+    private final Person person;
+    private final String description;
+
+    /**
+     * Constructs a {@code PersonEvent}
+     *
+     * @param localDate Date of the event.
+     * @param person Person whom the event belongs to.
+     * @param description A description of the event.
+     */
+    public PersonEvent(LocalDate localDate, Person person, String description) {
+        this.date = localDate.getDayOfMonth();
+        this.month = localDate.getMonthValue();
+        this.person = person;
+        this.description = description;
+    }
+
+    public int getDate() {
+        return date;
+    }
+
+    public int getMonth() {
+        return month;
+    }
+
+    public Person getPerson() {
+        return person;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
```

Comment:
Should have a `toUi` method here to do the formatting using the `LocalDate` formatter methods.